src/main/scala/org/scalameter/execution/LocalExecutor.scala

```diff
@@ -27,6 +27,7 @@ import utils.Tree
  *  the young generation.
  */
 class LocalExecutor(val warmer: Warmer, val aggregator: Aggregator, val measurer: Measurer) extends Executor {
+  require(!measurer.isInstanceOf[InvocationCount], "InvocationCount measurers should be run using SeparateJvmsExecutor")
```

Comment:
This is a bit ad-hoc. I would add a method `usesInstrumentedClasspath` to `Measurer`, which defaults to `false`.